Gemfile.lock

```diff
@@ -105,14 +104,9 @@ GEM
       rubocop (~> 0.51.0)
       rubocop-rspec (~> 1.19.0)
       scss_lint
-    govuk_ab_testing (2.4.1)
```

Comment:
I guess we don't need this any more (it was probably us that added it)

Reply:
No, there's nothing using it at the moment.